app/src/main/java/io/stanwood/bitrise/data/net/BitriseService.kt

```diff
@@ -49,4 +50,10 @@ interface BitriseService {
             @Path("APP-SLUG") appSlug: String,
             @Path("BUILD-SLUG") buildSlug: String,
             @Path("ARTIFACT-SLUG") artifactSlug: String): Deferred<Response<Artifact>>
+
+    @Suppress("EnumEntryName")
+    enum class SortBy {
+        last_build_at,
```

Comment:
Non canonical naming. @SerializedName can be used instead.

Reply:
Doesn't work here unfortunately. :(